### DISCUSSION TITLE
Flashback Notes

### DISCUSSION THREAD

**Discussion by damionx7** [damionx7/Zettel-Notes-Documentation#117] [Feature Requests]
[TB]: The idea is stolen from [Bebob](https://apps.apple.com/us/app/bebop-quick-notes/id6477824795?ref=jackcheng.com).
Why: If the user thinks of something while editing/creating a note that he or she would like to add to the most recently created note, he or she must first take the detour via the notes list. By then his idea might be gone again 😉.
What: The user can switch between the most recently created (or edited) notes using a button (like a reporter's notepad).
How: In editor mode, the user can "jump around" between the most recent notes (e.g. between the notes of a day or a certain number[^1]) by pressing a button. By pressing a button, he then opens the previous note in editor mode until the cycle is completed once.
[^1]: The user could set the number in the settings, for example.
Alternatives:

    A forward/backward function would also be conceivable to jump around between notes
    Or a function similar to ALT+Tab on Windows

**Comment by Lxtharia** [comment]
Hello, writing my suggestions here, because its quite similar.
What I really miss is exactly this ability, to switch to any note without having to navigate via the file list.
I'm coming from obsidian which has the quickswitch feature:

https://github.com/user-attachments/assets/415bbf51-efcc-49fa-907a-648292b487bb

(I would try to implement this as a plugin but I couldn't quite get the example plugin to compile yet 😅)

**Comment by damionx7** [comment]
Hey, You can switch from the right navigation drawer. Open it by clicking of the drawer button in note viewer. Then click on search button in bottom left.

**Comment by Lxtharia** [comment]
Thank you! I barely use the viewer tbh, thats why i never found it probably.
But still some improvemeny suggestions:
- Adding the search as a option in the toolbar would save one or two clicks
- Alternarively beeing able to swipe from the right to open the drawer (even from edit mode)
- Have the most recent files at the top of the search

Would be a huge efficiency gain i'd argue
